controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java

```diff
@@ -230,6 +232,12 @@ void store(Position position) {
                        final String readerId,
                        final int index,
                        final CheckpointStore checkpointStore) {
+        Preconditions.checkNotNull(eventProcessorConfig);
```

Comment:
FYI, this method returns the argument, so you can compress these into something like
```
this.reader = Preconditions.checkNotNull(reader);
```
etc.